### PR TITLE
Add links to probe definitions

### PR DESCRIPTION
**Pull request by fbertsch**
My web-dev days are long gone. If this is an awful ugly way to do it please correct me.

### CODE REVIEW

File: new-pipeline/src/dashboards.js [hunk 1]
```diff
@@ -1045,3 +1045,11 @@ function updateOSs() {
     optionGroupLabel.addClass("all-selected");
   });
 }
+
+function getDescriptionWithLink(metric, channel, description) {
+  var metricUrlTitle = "View Definition"
+  var metricUrl = "https://georgf.github.io/fx-data-explorer/index.html?search=" + metric + "&searchtype=in_name&optout=false&channel=" + channel + "&constraint=is_in&version=any"
+  var descr = description === null ? metric : description
+
+  return descr + " <a href=\"" + metricUrl + "\" target=\"_blank\">" + metricUrlTitle + "</a>"
```
Comment (chutten) [inline new-pipeline/src/dashboards.js]: Didn't we _just_ remove the last `target=_blank` from the site like four days ago...?

**Review by chutten — COMMENTED** [review]
It doesn't look any worse than the code surrounding it...

File: new-pipeline/src/dashboards.js (hunk 1 above)
Comment (fbertsch) [reply new-pipeline/src/dashboards.js]: I know, but given that you have this plot open, do you really want to close it by clicking on the link? I sure wouldn't.

**Comment by fbertsch** [comment]
@chutten test it here: https://fbertsch.github.io/telemetry-dashboard/

**Comment by chutten** [comment]
How's it look if we linkify the description itself? Any better?

**Comment by fbertsch** [comment]
@chutten updated, it looks fine, but I just don't want to have to educate people about that link.

**Comment by chutten** [comment]
Maybe stick http://upload.wikimedia.org/wikipedia/commons/6/64/Icon_External_Link.png after it?

**Comment by chutten** [comment]
Or don't. I'm not sure what an attractive design would even look like, so I'm not sure why I'm bikeshedding here :(

**Comment by fbertsch** [comment]
@chutten actually I think that does it! Makes it very clear it's a link and that it opens in a new tab.

File: new-pipeline/src/dashboards.js [hunk 2]
```diff
@@ -1045,3 +1045,12 @@ function updateOSs() {
     optionGroupLabel.addClass("all-selected");
   });
 }
+
+function getDescriptionWithLink(metric, channel, description) {
+  var metricUrlTitle = "View Definition"
```
Comment (chutten) [inline new-pipeline/src/dashboards.js]: Unused

**Review by chutten — COMMENTED** [review]
Ack, you changed it from under me.

**Review by chutten — APPROVED** [review]

**Pull request merged**